### PR TITLE
Adding toggleCheck action in attendees

### LINKED ISSUE #1986: Not possible to check-in attendees
The checkin attendee functionality is not working, e.g. here https://eventyay.com/events/3689bc98/tickets/attendees/completed

screenshot from 2019-01-26 22-16-28

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
It adds the functionality to toggle between checked in status of an attendee between `true` and `false`

#### Changes proposed in this pull request:

- Adding `toggleCheck` action in attendees to toggle isCheckedIn field
- Modifying attendees/cell-action template along the lines of sessions/cell-buttons

Fixes #1986

### CODE REVIEW

**Comment by uds5501** [comment]
@mariobehling @CosmicCoder96 
Please see if the workflow is good enough.

File: app/controllers/events/view/tickets/attendees/list.js [hunk 1]
```diff
@@ -34,5 +34,17 @@ export default Controller.extend({
       template       : 'components/ui-table/cell/events/view/tickets/attendees/cell-action',
       disableSorting : true
     }
-  ]
+  ],
+  actions: {
+    toggleCheck(attendee) {
```
Comment (CosmicCoder96) [inline app/controllers/events/view/tickets/attendees/list.js]: Rename this action as `toggleCheckIn`, rest looks good.

Comment (uds5501) [reply app/controllers/events/view/tickets/attendees/list.js]: @CosmicCoder96 alright, will do

Comment (uds5501) [reply app/controllers/events/view/tickets/attendees/list.js]: @CosmicCoder96 alright, will do

Comment (uds5501) [reply app/controllers/events/view/tickets/attendees/list.js]: Request acknowledged, kindly review @CosmicCoder96

**Comment by uds5501** [comment]
@CosmicCoder96 shall I squash the commits ?

**Pull request merged**